src/dmd/argtypes.d

```diff
@@ -29,6 +29,7 @@ import dmd.visitor;
  * Returns:
  *      tuple of types, each element can be passed in a register.
  *      A tuple of zero length means the type cannot be passed/returned in registers.
+ *      null indicates a `void`.
```

Comment:
But none of the callers of toArgTypes check for `null` as far as I recall.

Reply:
Good reason to document it!